locales/en/topic.json

```diff
@@ -1,4 +1,20 @@
 {
   "create_topic": "Create topic",
-  "show_all_options": "Show all available options"
+  "empty_topics_body": "Create a topic by clicking the button below to get started.",
+  "empty_topics_title": "No topics yet",
+  "input_field_invalid_message": "Valid characters include letters from a to z, numbers from 0 to 9, and hyphens ( - ).",
+  "partitions": "$t(create-topic:partitions)",
+  "retention_size": "Retention size",
+  "retention_time": "Retention time",
+  "show_all_options": "Show all available options",
+  "table": {
+    "actions": {
+      "delete": "Delete",
+      "edit": "Edit"
```

Comment:
As part of refactoring the Kafka table view, we updated the labels in the action menu based on usability test findings. Can we do the same for the actions in this menu?

The updates would be:
* Edit 👉 Edit topic configuration
* Delete 👉 Delete topic